Dockerfile

```diff
@@ -0,0 +1,39 @@
+# FROM ubuntu-upstart:precise
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage:0.9.9
+
+# Install pre-pre-requisites.
+RUN apt-get update
```

Comment:
🔢 Can we chain commands instead? e.g. `RUN apt-get update && apt-get install -q -y wget #ca-certificates`

Seeing I've been working with Docker on a client project I've been "levelling up" on Docker training today and the consensus I've seen is that Docker folks tend to prefer chaining as it leaves less overhead.